google-cloud-firestore/src/main/java/com/google/cloud/firestore/telemetry/BuiltinMetricsConstants.java

```diff
@@ -0,0 +1,154 @@
+/*
+ * Copyright 2024 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *       http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.google.cloud.firestore.telemetry;
+
+import com.google.api.gax.tracing.OpenTelemetryMetricsRecorder;
+import com.google.common.collect.ImmutableMap;
+import com.google.common.collect.ImmutableSet;
+import io.opentelemetry.api.common.AttributeKey;
+import io.opentelemetry.sdk.metrics.InstrumentSelector;
+import io.opentelemetry.sdk.metrics.View;
+import java.util.Map;
+import java.util.Set;
+import java.util.stream.Collectors;
+
+public class BuiltinMetricsConstants {
+
+  static final String FIRESTORE_LIBRARY_NAME = "firestore_java";
```

Comment:
should this be `java-firestore`? (the name of this repo)
or maybe `com.google.cloud.firestore` (the name of the java package)
Where will this name show up?

Reply:
it is the attribute value that will show up in each metrics data like this: {library_name: "java_fiestore", library_version:"3.25.1", method:....}